testsuite/forge/src/interface/aptos.rs

```diff
@@ -332,20 +332,12 @@ pub async fn reconfig(
     transaction_factory: &TransactionFactory,
     root_account: Arc<LocalAccount>,
 ) -> State {
-    let aptos_version = client.get_aptos_version().await.unwrap();
-    let current = aptos_version.into_inner();
-    let current_version = *current.major.inner();
     let txns = {
-        vec![
-            root_account.sign_with_transaction_builder(transaction_factory.clone().payload(
-                aptos_stdlib::version_set_for_next_epoch(current_version + 1),
-            )),
```

Comment:
this being called from multiple threads can cause issue because it reads the current version from onchain and two threads can generate txn to bump the version to the same number, and the second will fail.
I don't see why we need to make the call at all now that we have aptos_governance_force_end_epoch_test_only()

@igor-aptos  @zekun000 ?

Reply:
don't you have to change `fn submit_and_wait_reconfig()` accordingly then?

Reply:
sorry, no you don't...